### ISSUE TITLE
Add a taskbar icon

### ISSUE THREAD

**Issue by prmr** [prmr/JetUML#57]
Set up a manifest for the deployable jar so it shows a desktop/taskbar icon (a small version of the application icon).

**Comment by prmr** [comment]
http://stackoverflow.com/questions/209812/how-do-i-change-the-default-application-icon-in-java

**Comment by JoelChev** [comment]
Alright, I have added a taskbar icon in the branch Issue0057. Please let me know if this looks ok, or if you want it to be different. It seems to be working fine on my end, but I would like someone else to look at it before I merge this branch into master.

**Comment by prmr** [comment]
This doesn't work in deployed mode (exported as a jar). Any change involving resources should be tested in deployed mode as well. The issue is that you need to use 
`getClass().getClassLoader().getResource(...)`. Please see the comments for Issue #56.

**Comment by prmr** [comment]
Also, the application fails to load if the resource isn't found. This seems a bit brittle. It would be better to make this path recoverable, so that if the resource isn't found that application can launch anyways without the taskbar icon.

**Comment by JoelChev** [comment]
Thank you for the catch! I just made the path to the app.icon file relative and added a new folder to the icon folder to hold the image. I tested it in deployed mode and it seems to be working now. Can you please confirm that the changes worked? Thanks!

**Comment by prmr** [comment]
The change broke the About menu. When changing the path to resources, it's a good idea to do a reference check.

**Comment by JoelChev** [comment]
Good catch again. Thanks for looking into it. I've changed the code again, so the icon should properly display now. Thanks!
